### DISCUSSION TITLE
backfill API support making the success state backfill task_instance to rerun

### DISCUSSION THREAD

**Discussion by yyhecust** [apache/airflow#27752] [Ideas]
Description
Like how we are able to rerun backfill instances of failed status using airflow API, we need a feature where we can rerun backfill instances of success status. This feature is needed for us.

Use case / motivation
Sometimes, we need to make some changes to the DAG files so that require to rerun the backfill instances to get new data. But airflow only offers rerun of failed tasks for the backfill function, and we want it will also support auto-rerun all the succeed tasks for the backfill date range instead of throwing exceptions.

Are you willing to submit a PR?
Yes, I'm willing to.

Related Issues

**Comment by uranusjr** [comment]
I kind of feel that if your change to the DAG is sufficient to need to rerun backfills, you probably should make it a new DAG instead.
